feat(changelog): billing-added-discover-our-new-compute-savings-plan-off 2024-09-27 (#3767)
scaleway/docs-content: changelog/september2024/2024-09-27-billing-added-discover-our-new-compute-savings-plan-off.mdx

---

> title: Discover our New Compute Savings Plan offer

Reviewer: ```suggestion
title: Discover our new Compute Savings Plan offer
```

status: added
author:
    fullname: ''
    url: 'https://slack.scaleway.com'
date: 2024-09-27
category: billing
product: billing
---

> Scaleway’s Savings Plan is a commitment-based discount program that provides more cost-effective billing compared to the pay-as-you-go model. As for now, this offer applies to our Instance products [(see the details)](https://www.scaleway.com/en/docs/console/billing/additional-content/understanding-savings-plans/#compute-savings-plan) in your Billing space, within the “Cost optimization” tab.

Reviewer: ```suggestion
Scaleway’s Savings Plan is a commitment-based discount program that provides more cost-effective billing compared to the pay-as-you-go model. Currently, this offer can be used with [Instance products.](/console/billing/additional-content/understanding-savings-plans/#compute-savings-plan) Access Savings Plans in the [Scaleay console](https://console.scaleway.com) in the **Cost optimization** tab of your Billing space.
```

> Read [our dedicated documentation](https://www.scaleway.com/en/docs/console/billing/additional-content/understanding-savings-plans/) for more information and to purchase your first Savings Plan.

Reviewer: ```suggestion
Read [our dedicated documentation](/console/billing/additional-content/understanding-savings-plans/) for more information and to purchase your first Savings Plan.
```
